x/evm/types/tx_args.go

```diff
@@ -35,6 +35,7 @@ import (
 type TransactionArgs struct {
 	From                 *common.Address `json:"from"`
 	To                   *common.Address `json:"to"`
+	OnBehalf             *common.Address `json:"on_behalf"`
```

Comment:
```suggestion
	OnBehalf             *common.Address `json:"onBehalf"`
```
Naming doesn't match with others